code-challenges.test.js

```diff
@@ -30,16 +51,36 @@ const secretCodeWord3 = "Eccentric"
 const fruitArray = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach", "Kiwi"]
 
 const letterA = "a"
+
 // Expected output: ["Mango", "Apricot", "Peach"]
 const letterE = "e"
 // Expected output: ["Cherry", "Blueberry", "Peach"]
+expect(findMe(fruitArray, letterA)).toEqual(["Mango", "Apricot", "Peach"])
+    expect(findMe(fruitArray, letterE)).toEqual(["Cherry", "Blueberry", "Peach"])
 
 // b) Create the function that makes the test pass.
+// params will be the array and letters inside it
+// it will  make letters lowercase then filter it
+// if it includes a wanted letter then it will return it
+const wordBundle = (wordArr, letter) => {
+    letter = letter.toLowerCase()
+    let output = wordArr.filter((word => {
+        if (word.toLowerCase().includes(letter)) {
+
+            //Return the current word 
+            return word
+        }
+    }))
+    //Return result iteration after completion
+    return result
+
```

Comment:
Great job with the logic, but you return result instead of output.  Be sure to run your code before you submit.  Also, your test is incomplete.